package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "wavs-middleware",
+  "version": "1.0.0",
```

Comment:
Please use "0.4.0-alpha.1" as that is the current version.
We should track WAVS tags, probably "0.4.0-beta" soon

Reply:
resolved: e394023a6fd46e1aeac74e8cc7c8ff197d9ec564